### PR TITLE
[test/plugin] merge undertow scenario

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

Merge ```undertow-scenario``` and ```undertow-routing-scenario``` , save about 10 minutes.

### CODE REVIEW

File: test/plugin/run.sh [hunk 1]
```diff
@@ -143,9 +143,9 @@ test -z "$scenario_name" && exitWithMessage "Missing value for the scenario argu
 
 if [[ ! -d ${agent_home} ]]; then
     echo "[WARN] SkyWalking Agent not exists"
-    ${mvnw} -f ${home}/../../pom.xml -Pagent -DskipTests -DBUILD_NO=${BUILD_NO:=local} clean package
+    ${mvnw} -f ${home}/../../pom.xml -Pagent -DskipTests clean package
 fi
-[[ "$force_build" == "on" ]] && ${mvnw} -f ${home}/pom.xml clean package -DskipTests docker:build
+[[ "$force_build" == "on" ]] && ${mvnw} -f ${home}/pom.xml clean package -DskipTests -DBUILD_NO=${BUILD_NO:=local} docker:build
```
Comment (arugal) [inline test/plugin/run.sh]: FYI @dmsolr

**Comment by wu-sheng** [comment]
You need to update https://github.com/apache/skywalking/blob/master/docs/en/guides/Plugin-test.md#the-elapsed-time-list-of-plugins by following the CI result.

**Comment by wu-sheng** [comment]
@arugal The doc file has been updated, please follow the change.

**Pull request merged**